tests/test_arg_recorder.py

```diff
@@ -100,7 +100,7 @@ def test_simplify2(self):
         # test that we get the same tree sequence by doing tree_sequence
         # and simplify -> tree_sequence
         records = ftprime.ARGrecorder(ts=self.init_ts, node_ids=self.init_map,
-                                      sequence_length=1.0)
+                                      sequence_length=0.5)
```

Comment:
The error I get from this is
```
E       _msprime.LibraryError: Right coordinate > sequence length.
```
and is entirely sensible, because the sequence length of `init_ts` is greater than 0.5.

I guess we can add a check for this if you want, although the error is already pretty informative?

Reply:
There would actually be use cases for this - e.g. intiializing with the empty tree sequence and passing in a longer sequence_length - which is why I didn't put that error check before - but not *good* use cases.